src/components/application_manager/test/mock_message_helper.cc

```diff
@@ -115,9 +115,12 @@ mobile_api::HMILevel::eType MessageHelper::StringToHMILevel(
   return MockMessageHelper::message_helper_mock()->StringToHMILevel(hmi_level);
 }
 
-smart_objects::SmartObjectSPtr CreateDeviceListSO(
-    const connection_handler::DeviceMap& devices) {
-  return MockMessageHelper::message_helper_mock()->CreateDeviceListSO(devices);
+smart_objects::SmartObjectSPtr MessageHelper::CreateDeviceListSO(
```

Comment:
@pold500 I think that you accidently deleted realization for this method (CreateDeviceListSO) with only one parameter, or the method with one parameter is unnecessary? In that case, shouldn't this method be deleted from mock_message_helper.h too?